kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -76,7 +76,7 @@ static String requireIsDnsLabel(String string, boolean rfc1035, String message)
         return string;
     }
 
-    static <O extends HasMetadata> OwnerReference ownerReferenceTo(O owner) {
+    public static <O extends HasMetadata> OwnerReference ownerReferenceTo(O owner) {
```

Comment:
I think the fact that this is creating a new OR (rather than getting an OR from its parameter; the rest of the functions in this class are getters) is an important path of this method. So `newOwnerReferenceTo()` or `makeOwnerReferenceTo()` might be better names.